backend/wmg/data/schema.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import tiledb
+
+cube_indexed_dims = [
+    "gene_term_id",
```

Comment:
hmm, should this be `gene_ontology_term_id`?  Or maybe eliminate all `ontology_` infixes from other schema names? It's verbose, but it is also clearer...

Reply:
I like the ontology_ infixes, I think they add clarity

Reply:
reinstated!